src/style.css

```diff
@@ -1,3 +1,81 @@
+* {
+  margin: 0;
+  padding: 0;
+
+  --hover-color: #00acdc;
```

Comment:
Create a variable with root selector

Reply:
not fixed 
<img width="897" alt="image" src="https://github.com/user-attachments/assets/809bd62e-b56f-4284-bc5f-cff7fd4b6212">